mypy.ini

```diff
@@ -1,6 +1,6 @@
 [mypy]
 python_version = 3.8
-plugins = edb.tools.mypy.plugin
+plugins = edb/tools/mypy/plugin.py
```

Comment:
Would this work if I run mypy from a different working directory?

Reply:
Somewhat surprisingly, yes